Task_1/Manage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task_1
+{
+    public class Manage
```

Comment:
Classes should have names that describes what it does, not just verbs.
e.g. DeviceManager.